### ISSUE TITLE
Can we please get some communication??

### ISSUE THREAD

**Issue by wesrt36** [sologenic/sologenic-expo-wallet#453]
We did everything right. We believed when you said there was no need to set a trust line. We set it when we were told, and still nothing. We moved money and many of us lost money. We understand this is free money and you didn't have to do this, but when people lose money by trusting what you say when they could have left it alone and been safer, it is wrong to do what you guys are doing. 

Please at the very least keep us updated as to what's happening or tell us what we're doing wrong, because according to the information you are providing, we aren't.

And it makes one wonder if you have less-than-altruistic intentions getting large amounts of people to move large amounts of money to your platform just to leave them in the dark.

**Comment by Kwizlyte12** [comment]
I agree I been trying to send my XRP back to the address I sent it from, all I keep getting is my destination tag is incorrect which is not try!! Everything is correct I quadruple checked!! I’m beginning to think it’s a scam and we all lost!!!

**Comment by Martano11** [comment]
Hey Guys. I transferred my Xrp back to an exchange as soon as the solo airdrop did not come through. They originally told us a trust line was not required if xrp held in their solo dex app. Then trying to set up a trust line was near on impossible and in the end I gave up. My advice, for what it’s worth, is get your xrp out of solo ASAP.

**Comment by Fladder1** [comment]
To  wesrt36    Fladder1 here, I completely agree with what you are saying. Thanks to you I was able to set up the trustline for the Solo 2nd drop.
And then with almost no time to react "they"
finally released the issuer letter combination for the Core drop. And still I didn't receive Solo nor Core. This whole holobulu was humbug in my eyes.
